<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>711</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
   <widget class="QGroupBox" name="groupBox_1">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Цвет отсекателя</string>
    </property>
    <widget class="QComboBox" name="MainPenColor_comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>80</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Цвет отрезков</string>
    </property>
    <widget class="QComboBox" name="SemiPenColor_comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>150</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Цвет результата</string>
    </property>
    <widget class="QComboBox" name="backgroundColor_comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="clean_Btn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>590</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить многоугольник</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fill_Btn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>540</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Отсечь</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>310</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Ввод точек</string>
    </property>
    <widget class="QLineEdit" name="X_Edit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>350</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Y">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y = </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_X">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X = </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="Y_Edit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>350</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_point_Btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить точку</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>220</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Любое изменение цветов
приведёт к очистке</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>260</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Выбор ввода</string>
    </property>
    <widget class="QRadioButton" name="Cut_mode_radioButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Отсекатель</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Edge_mode_radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Многоугольник</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="lock_Btn">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>490</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Замкнуть отсекатель</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lock_Btn_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>440</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Замкнуть многоугольник</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clean_Btn_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>640</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить отсекатель</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>987</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
